integration-test-common/src/test/java/org/apache/gravitino/integration/test/container/HiveContainer.java

```diff
@@ -49,6 +49,7 @@ public class HiveContainer extends BaseContainer {
   public static final String HIVE_RUNTIME_VERSION = "HIVE_RUNTIME_VERSION";
   public static final String HIVE2 = "hive2"; // The Hive container default version
   public static final String HIVE3 = "hive3";
+  public static final String ENABLE_AUTHORIZATION = "ENABLE_JDBC_AUTHORIZATION";
```

Comment:
I think it better to change `ENABLE_AUTHORIZATION` to `ENABLE_SQL_BASE_AUTHORIZATION`?

Reply:
OK, I can create a new Hive image.

Reply:
OK. I will change this.